atom/browser/api/atom_api_app.cc

```diff
@@ -1063,8 +1065,9 @@ void App::ImportCertificate(const base::DictionaryValue& options,
         base::Value::ToUniquePtrValue(options.Clone()));
     CertificateManagerModel::Create(
         browser_context.get(),
-        base::Bind(&App::OnCertificateManagerModelCreated,
-                   base::Unretained(this), base::Passed(&copy), callback));
+        base::BindRepeating(&App::OnCertificateManagerModelCreated,
+                            base::Unretained(this), base::Passed(&copy),
```

Comment:
can you try `std::move` instead of `base::Passed` here.

Reply:
Looks like the change needs to be reverted, cannot forward in repeating variation.